README.md

```diff
@@ -210,7 +210,14 @@ enough for most real-life use cases such as receipts. Some news sites may need
 even 2GB of RAM.*
 
 
-#### 2. Local development
+#### 2. Run in Docker
+
+* `docker build -t example/url-to-pdf .`
```

Comment:
I've been using docker quite a lot but forget its flags and naming conventions. Why is the name `example/url-to-pdf`? Can one run these commands in the root of this project and it just works? (assuming one has Docker installed)

Reply:
`-t` is the tag name of the docker image. I think the command should works in the project root directory. 

I think we can provide a `docker-compose.yml` for the ease of bootstrap? Just `docker-compose up` and the image will be built and up.

Reply:
Would we be having a version provided by the repo? So  I can just run `docker run alvarcarto/url-to-pdf-api` ?